packages/e2e.sandbox.catchall/src/index.ts

```diff
@@ -33,7 +32,6 @@ const testSubscriptions = false;
 
 async function runTests() {
   try {
-    await runFoundryPlatformApiTest();
```

Comment:
Why'd we delete this?

Reply:
The apis that test used are no longer generated any more. We have other tests for the platform api so no loss.